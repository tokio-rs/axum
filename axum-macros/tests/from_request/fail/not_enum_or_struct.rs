use axum_macros::FromRequest;

#[derive(FromRequest)]
union Extractor {}

fn main() {}
