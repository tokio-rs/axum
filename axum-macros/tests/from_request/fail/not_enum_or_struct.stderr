error: expected `struct` or `enum`
 --> tests/from_request/fail/not_enum_or_struct.rs:4:1
  |
4 | union Extractor {}
  | ^^^^^^^^^^^^^^^^^^

error: unions cannot have zero fields
 --> tests/from_request/fail/not_enum_or_struct.rs:4:1
  |
4 | union Extractor {}
  | ^^^^^^^^^^^^^^^^^^
